coalib/bears/Bear.py

```diff
@@ -249,8 +250,19 @@ def execute(self, *args, **kwargs):
             result = self.run_bear_from_section(args, kwargs)
             return [] if result is None else list(result)
         except (Exception, SystemExit):
-            self.warn('Bear {} failed to run. Take a look at debug messages'
-                      ' (`-V`) for further information.'.format(name))
+            try:
+                if self.kind() == BEAR_KIND.GLOBAL:
+                    self.warn('Bear {} failed to run. Take a look '
+                              'at debug messages (`-V`) for further '
+                              'information.'.format(name))
+                else:
+                    self.warn('Bear {} failed to run on file {}. Take a look '
+                              'at debug messages (`-V`) for further '
+                              'information.'.format(name, args[0]))
+            except (Exception, SystemExit):
+                self.warn('Bear {} failed to run. Take a look '
+                          'at debug messages (`-V`) for further '
+                          'information.'.format(name))
```

Comment:
Completely omitting the second `try catch` inside the `except` block will result in failure of some tests since the bear [here](https://github.com/coala/coala/blob/master/tests/bears/BearTest.py#L16) doesn't have a kind. So while executing [this](https://github.com/coala/coala/blob/master/tests/bears/BearTest.py#L121) test, when it enters the if condition ie. `self.kind() == BEAR_KIND.LOCAL`, it will throw an exception IIUC.